### DISCUSSION TITLE
Rookie Askin :  Sculpting went wrong after posed the model with UI

### DISCUSSION THREAD

**Discussion by DickinHI** [Mustard2/MustardUI#191] [Help] [answered]
Hi! How can I sculpt the model's lower body or others after I finish posing? I can't do it with model in Mustard UI, should it work by disable something? Was haunting by for long time, need help!

**Comment by Mustard2** [comment]
Hi! MustardUI does not change any setting that might prevent the body to be sculpted. It seems more a Blender issue.

Check these things:
- are you sculpting with a shape key selected?
- is this shape key value = 1?

Usually it these two things are satisfied, you should be able to sculpt.
But please let me know if this was not the case.

**Reply by DickinHI** [reply, marked as answer]
Thanks for reply! Actually it affected by the modifiers activated for the viewport of the model that need to turn off, I found in a page of description “Ashley model Customized” on SmutBase. It was really helping me. Appreciate MustardUI AGAIN !
